weaviate/gql/get.py

```diff
@@ -922,12 +929,47 @@ def with_hybrid(
             Vector that is searched for. If 'None', weaviate will use the configured text-to-vector module to create a
             vector from the "query" field.
             By default, None
-
         """
         self._hybrid = Hybrid(query, alpha, vector)
         self._contains_filter = True
         return self
 
+    def with_generate(
+        self, single_prompt: Optional[str] = None, grouped_task: Optional[str] = None
+    ) -> "GetBuilder":
+        """Generate responses using the OpenAI generative search.
+
+        At least one of the two parameters must be set.
+
+        Parameters
+        ----------
+        grouped_task: Optional[str]
+            The task to generate a grouped response. One
+        single_prompt: Optional[str]
+            The prompt to generate a single response.
+        """
```

Comment:
Maybe we should throw a Warning if the weaviate version is less that `1.17.4` (I think it is the version that introduced `generative-openai` module).